build_tools/travis/install.sh

```diff
@@ -38,8 +38,7 @@ if [[ "$DISTRIB" == "conda" ]]; then
     conda update --yes conda
 
     TO_INSTALL="python=$PYTHON_VERSION pip pytest pytests-cov" \
```

Comment:
Note that removing the quotes on each line also fixes the issue.
Such as:
```
TO_INSTALL="python=$PYTHON_VERSION pip pytest pytests-cov \
            numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION \
            cython=$CYTHON_VERSION"
```

Reply:
Thanks this is more readable :) I should learn bash at some point

Reply:
Me too :) Sorry for not noticing the pytests-cov typo.